src/itemActionTitle/ItemActionTitle.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import cc from 'classcat'
+
+import { Item } from '../_internals/item'
+import { color } from '../_utils/branding'
+import { useFocusVisible } from '../_utils/focusVisibleProvider/useFocusVisible'
+import { A11yProps, pickA11yProps } from '../_utils/interfaces'
+import { NormalizeProps } from '../layout/layoutNormalizer'
+
+export type ItemActionTitleProps = NormalizeProps &
+  A11yProps &
+  Readonly<{
+    className?: string
+    href?: string | JSX.Element
+    titleLabel?: string
```

Comment:
I think we should use `string | JSX.Element` in case we need to wrap the text with a `span` for instance.

Reply:
Done